test/combinators.spec.ts

```diff
@@ -1,4 +1,3 @@
-import 'mocha';
```

Comment:
How does this work?

Reply:
The `mocha` runner pre-loads itself into the globals, and the typescript config is told to be aware of that fact for code validation.  All our repos work like this :)